langchain4j-oracle-database/README.md

```diff
@@ -14,3 +14,11 @@ This module implements `EmbeddingStore` using Oracle Database.
 ```
 
 ## Configuration
+
+## Running the Test Suite
+By default, integration tests will
+[run a docker image of Oracle Database using TestContainers](https://java.testcontainers.org/modules/databases/oraclefree/).
+Alternatively, the tests can connect to an Oracle Database if the following environment variables are configured:
+- ORACLE_JDBC_URL : Set to an [Oracle JDBC URL](https://docs.oracle.com/en/database/oracle/oracle-database/23/jjdbc/data-sources-and-URLs.html#GUID-C4F2CA86-0F68-400C-95DA-30171C9FB8F0), such as `jdbc:oracle:thin@example:1521/serviceName`
+- ORACLE_JDBC_USER : Set to the name of a database user.
```

Comment:
Can we make user/password optional? (we need that for the config providers and in general it can go in the URL).

Reply:
Already optional. I'm updating the readme in a different branch.